### DISCUSSION TITLE
Add an ability to visualize storage contents

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5751] [Feature Requests]
This request is asking for the ability to open a storage node and see all the preparations in it as well as expand the storage node and see the preparations linked in a visual hierarchy. 

**Requested By:** Northestern

**Comment by melton-jason** [comment]
Can you expand further on this? 
I'm assuming the desired implementation of this feature would be the equivalent of what we have for the Taxon tree: to see how many records reference the Tree Node and provide the ability to run a query to find all of those records

**Comment by grantfitzsimmons** [comment]
Edited the original post.
